frontend/src/metabase/selectors/settings.ts

```diff
@@ -108,11 +109,9 @@ const getUtmSource = (features: TokenFeatures) => {
   }
 };
 
-const getTokenFeatures = (state: State) => getSetting(state, "token-features");
-
 export const getIsPaidPlan = createSelector(
-  getTokenFeatures,
-  (features: TokenFeatures) => {
-    return features.sso || features.hosting;
+  (state: State) => getSetting(state, "token-status"),
+  (token: TokenStatus | null) => {
+    return token != null && token.valid;
```

Comment:
nit: please use `tokenStatus` so it's consistent with other places e.g.
https://github.com/metabase/metabase/blob/c6607cc3b2824eec6e59977c627d021a2634e92b/enterprise/frontend/src/metabase-enterprise/license/components/LicenseAndBillingSettings/LicenseAndBillingSettings.tsx#L54

And `token` could also be easily mistaken as the token string.

Reply:
Agree, fixed